### ISSUE TITLE
onesync infinity

### ISSUE THREAD

**Issue by alexraja98** [PetrisGR/FiveM-BriefcaseMission#7]
can you help me with the onesync infinity with this script

**Comment by PetrisGR** [comment]
Hey, I can assist you here. What's the problem with Onesync Infinity?

**Comment by alexraja98** [comment]
The briefcase is showing to everyone without taking the briefcase at all mainly for police and ambulance job having this problem because they emergency blip .. How to make this solve

**Comment by PetrisGR** [comment]
That's not actually a problem. Briefcase is shown to everyone once revealed like the Hostile Takeover mission from GTA.

**Comment by alexraja98** [comment]
> That's not actually a problem. Briefcase is shown to everyone once revealed like the Hostile Takeover mission from GTA.

Without taking a briefcase itself it shows that I have a briefcase go and deliver to the delivery point

**Comment by PetrisGR** [comment]
Could you record a video with the issue?

**Comment by alexraja98** [comment]
> Could you record a video with the issue?
> 
> 

Yes I can https://youtube.com/clip/UgkxGl02V8VIl0Hu5zzmryLf3RSONe_wVHfj please check this

**Comment by PetrisGR** [comment]
Thank you for the clip, this seems a major problem. The problem will be under investigation. If you find any other issue, please let me know!

**Comment by PetrisGR** [comment]
Did that happen to all players or did only the streamer just "collected the briefcase" without collecting it?

**Comment by alexraja98** [comment]
> Did that happen to all players or did only the streamer just "collected the briefcase" without collecting it?

To all players but it depends upon the time . But mostly all players have faced this issuses

**Comment by alexraja98** [comment]
> Thank you for the clip, this seems a major problem. The problem will be under investigation. If you find any other issue, please let me know!

Yes sure that's for your reply I hope you can fix this issue soon

**Comment by Legacy-TacticalGamingInteractive** [comment]
I can confirm I am having this issue too
We did the mission today with 100 people on server

The person would take the briefcase for a few minutes then it would suddenly disappear from their hand and be back at the spawn location again and no one else could pick it up. Then time would expire. Mission cannot be completed.

**Comment by PetrisGR** [comment]
OK guys thank you for letting me know. The issue is being investigated.

**Comment by alexraja98** [comment]
> OK guys thank you for letting me know. The issue is being investigated.

When it will be resolved bro

**Comment by PetrisGR** [comment]
I will let you know when it's resolved. There will be a new version to download.

**Comment by Legacy-TacticalGamingInteractive** [comment]
any luck

**Comment by Legacy-TacticalGamingInteractive** [comment]
I also suggest using `PlayerPedId()` instead of `GetPlayerPed()` as its much faster

**Comment by alexraja98** [comment]
Any update bro
